ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -1201,6 +1201,43 @@ func TestMemReservationHigherThanMemLimit(t *testing.T) {
 	assert.EqualError(t, err, "mem_limit must be greater than mem_reservation")
 }
 
+func TestConvertToTaskDefinitionWithECSParams_PIDandIPC(t *testing.T) {
```

Comment:
can you add a test somewhere in this file to ensure that if `pid` or `ipc` aren't set they don't show up in the resulting task def? Maybe just add a couple `assert.Empty()` statements to the basic `TestConvertToTaskDefinition` and `TestConvertToTaskDefinition` tests?

I know right now there's not a precedence for testing the negative case(s) for specific fields, but we should think about how to assert on the negative/absent case for fields where it matters.

Reply:
Done